submissions/aymkin/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,276 @@
+/* NAV START */
+.container {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 340px;
+    position: absolute;
+    right: 10px;
+    top: -10px;
+}
+
+.element {
+    margin: 5px;
+}
+
+.tiel {
+    background-image: url(img/tile.png);
+    width: 32px;
+    height: 32px;
+}
+
+.icon {
+    width: 32px;
+    height: 32px;
+    border-radius: 50%;
+    background-image: url(img/account.png);
+}
+
+.element a {
+    text-decoration: none;
+    color: #111;
+    font-size: 1em;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+li {
+    list-style: none;
+    text-transform: capitalize;
+}
+
+/* NAV END */
+
+/* popup-ul START*/
+.popup {
+    display: flex;
+    flex-wrap: wrap;
+    width: 320px;
+    height: 500px;
+    border: #ccc solid 1px;
+    border-radius: 2px;
+    position: relative;
+    top: 10px;
+    right: 250px;
+    display: none;
+    overflow-y: auto;
+}
+
+.popup-ul,
+.more-ul {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 28px;
+    padding-bottom: 0px;
+}
+
+.popup-li {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 2px;
+    width: 65px;
+    margin: 8px;
+    border: 1px solid transparent;
+}
+
+.popup-li:hover {
+    border: 1px solid #ddd;
+}
+
+.popup-a:focus {
+    border: 1px solid rgba(125, 125, 255, 0.5);
+    border-radius: 2px;
+}
+
+.item {
+    width: 58px;
+    height: 58px;
+    margin-bottom: 8px;
+    cursor: pointer;
+}
+
+.item-1 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -1450px;
+}
+
+.item-2 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -1314px;
+}
+
+.item-3 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -210px;
+}
+
+.item-4 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -554px;
+}
+
+.item-5 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -2347px;
+}
+
+.item-6 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -1036px;
+}
+
+.item-7 {
```

Comment:
Try to use more meaningful classnames(without numbers). It's easier to understand your css without looking into markup.

Reply:
done